README.md

```diff
@@ -165,34 +165,27 @@ Configuration information is specified in two forms:
 | `request_timeout_ms`      | `30000`             | Request Timeout                                                           |
 | `api_key_sources`         | `PARAM,HEADER,PATH` | Defines the sources (order of precendence) of the API key.                |
 
-### System Properties for TLS configuration for Http server
+### Env variables for TLS configuration for Http server
 To configure Transport Layer Security (TLS) for the HTTP server in an edge module, the following configuration parameters should be used.
 Parameters marked as Required are required only in case when ssl_enabled is set to true.
 
-| Property                          | Default           | Description                                                                                 |
-|-----------------------------------|-------------------|---------------------------------------------------------------------------------------------|
-| `http-server.ssl_enabled`         | `false`           | Set whether SSL/TLS is enabled for Vertx Http Server                                        |
-| `http-server.keystore_type`       | `NA`              | (Required). Set the type of the keystore. Common types include `JKS`, `PKCS12`, and `BCFKS` |
-| `http-server.keystore_provider`   | `NA`              | Set the provider name of the key store                                                      |
-| `http-server.keystore_path`       | `NA`              | (Required). Set the location of the keystore file in the local file system                  |
-| `http-server.keystore_password`   | `NA`              | (Required). Set the password for the keystore                                               |
-| `http-server.key_alias`           | `NA`              | Set the alias of the key within the keystore.                                               |
-| `http-server.key_alias_password`  | `NA`              | Optional param that points to a password of `key_alias` if it protected                     |
-
-### System Properties for TLS configuration for Web Client
+| Property                                             | Default           | Description                                                                                 |
+|------------------------------------------------------|-------------------|---------------------------------------------------------------------------------------------|
+| `SPRING_SSL_BUNDLE_JKS_WEB-SERVER_KEYSTORE_TYPE`     | `NA`              | (Required). Set the type of the keystore. Common types include `JKS`, `PKCS12`, and `BCFKS` |
+| `SPRING_SSL_BUNDLE_JKS_WEB-SERVER_KEYSTORE_PATH`     | `NA`              | (Required). Set the location of the keystore file in the local file system                  |
+| `SPRING_SSL_BUNDLE_JKS_WEB-SERVER_KEYSTORE_PASSWORD` | `NA`              | (Required). Set the password for the keystore                                               |
+
+### Env variables for TLS configuration for Web Client
```

Comment:
Small suggestion: please add one empty line after title to be consistent with other places